src/linker/configuration.py

```diff
@@ -156,8 +187,43 @@ def _load_input_data_paths(input_data_specification_path: Path) -> List[Path]:
         return file_list
 
     @staticmethod
-    def _get_spark_requests(environment: Dict[str, Union[Dict, str]]) -> Dict[str, Any]:
-        spark = environment.get("spark", {})
-        if spark and not "keep_alive" in spark:
-            spark["keep_alive"] = False
-        return spark
+    def _get_computing_environment(environment: Dict[Any, Any]) -> str:
+        if not "computing_environment" in environment:
+            value = DEFAULT_ENVIRONMENT["computing_environment"]
+            logger.info(f"Assigning default value for computing_environment: '{value}'")
+        else:
+            value = environment["computing_environment"]
+        return value
+
+    @staticmethod
+    def _get_container_engine(environment: Dict[Any, Any]) -> str:
+        if not "container_engine" in environment:
+            value = DEFAULT_ENVIRONMENT["container_engine"]
+            logger.info(f"Assigning default value for container_engine: '{value}'")
+        else:
+            value = environment["container_engine"]
+        return value
+
+    @staticmethod
+    def _get_requests(environment: Dict[Any, Any], key: str) -> Dict[Any, Any]:
```

Comment:
This seems fine enough. I think we ultimately should refactor this to just use ConfigTrees, but not until we get ConfigTree out of vivarium. Because of that, we shouldn't invest any more in trying to make this config stuff more robust, because we'll just end up reimplementing ConfigTree.